lms/models/course.py

```diff
@@ -18,6 +18,11 @@ class Course(BASE):
         primary_key=True,
     )
 
+    #: The ApplicationInstance that this access token belongs to.
+    application_instance = sa.orm.relationship(
+        "ApplicationInstance", back_populates="courses"
+    )
```

Comment:
Lets you do `course.application_instance` and again informs SQLAlchemy of how to properly generate SQL for related objects